CIScripts/Tests/VRouterAgentTests.ps1

```diff
@@ -398,12 +511,54 @@ function Test-VRouterAgentIntegration {
         Write-Host "===> PASSED: Test-Pkt0ReceivesTrafficAfterAgentIsStarted"
     }
 
-    Test-InitialPkt0Injection -Session $Session -TestConfiguration $TestConfiguration
-    Test-Pkt0RemainsInjectedAfterAgentStops -Session $Session -TestConfiguration $TestConfiguration
-    Test-OnePkt0ExistsAfterAgentIsRestarted -Session $Session -TestConfiguration $TestConfiguration
-    Test-Pkt0ReceivesTrafficAfterAgentIsStarted -Session $Session -TestConfiguration $TestConfiguration
-    Test-GatewayArpIsResolvedInAgent -Session $Session -TestConfiguration $TestConfiguration
+    function Test-SingleComputeNodePing {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+               [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+        Write-Host "===> Running: Test-SingleComputeNodePing"
+        Test-Ping -Session1 $Session -Session2 $Session -TestConfiguration $TestConfiguration -Container1Name "container1" -Container2Name "container2"
+        Write-Host "===> PASSED: Test-SingleComputeNodePing"
+    }
+
+    function Test-MultiComputeNodesPing {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session1,
+               [Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session2,
+               [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+        Write-Host "===> Running: Test-MultiComputeNodesPing"
+        Test-Ping -Session1 $Session1 -Session2 $Session2 -TestConfiguration $TestConfiguration -Container1Name "container1" -Container2Name "container2"
+        Write-Host "===> PASSED: Test-MultiComputeNodesPing"
+    }
+
+    $AgentIntegrationTestsTimeTracker.StepQuiet("Test-SingleComputeNodePing", {
```

Comment:
I think that it would be more logical to test ping at the end of the test suite because for example MultiComputeNodesPing needs pkt0 interface working so it should be tested first.